<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewer</class>
 <widget class="QMainWindow" name="viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="algos_layout">
     <item row="1" column="0">
      <widget class="QRadioButton" name="par_eq_but">
       <property name="text">
        <string>Parametric equation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="can_eq_but">
       <property name="text">
        <string>Canonical equation</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="midpoint_but">
       <property name="text">
        <string>Midpoint algo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="auto_but">
       <property name="text">
        <string>auto</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="bres_but">
       <property name="text">
        <string>Bresenham</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>261</width>
      <height>266</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="text_fields_layout">
     <item row="9" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>ellipse radius ver:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>center x: </string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QLineEdit" name="ellipse_radius_ver_text">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="center_y_text">
       <property name="text">
        <string>300</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="center_x_text">
       <property name="text">
        <string>550</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>circle radius:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>spectrum center x:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>spectrum center y:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLineEdit" name="circle_radius_text">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>ellipse radius hor:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="spectrum_center_x_text">
       <property name="text">
        <string>700</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="spectrum_center_y_text">
       <property name="text">
        <string>350</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>objects in spectrum:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLineEdit" name="ellipse_radius_hor_text">
       <property name="text">
        <string>200</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="obj_num_text">
       <property name="text">
        <string>30</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>center y:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>spectrum step:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="step_text">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="colors_layout">
     <item row="1" column="0">
      <widget class="QRadioButton" name="green_but">
       <property name="text">
        <string>green</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="blue_but">
       <property name="text">
        <string>gray</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="bacground_but">
       <property name="text">
        <string>background</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="red_but">
       <property name="text">
        <string>red</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="white_but">
       <property name="text">
        <string>white</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="yellow_but">
       <property name="text">
        <string>yellow</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="magenta_but">
       <property name="text">
        <string>magenta</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QRadioButton" name="cyan_but">
       <property name="text">
        <string>cyan</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>261</width>
      <height>282</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="7" column="0">
      <widget class="QPushButton" name="erase_but">
       <property name="text">
        <string>erase</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="draw_circle_but">
       <property name="text">
        <string>draw circle</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="circle_time_stat_but">
       <property name="text">
        <string>circle time statistics</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="draw_ellipse_but">
       <property name="text">
        <string>draw ellipse</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="draw_circle_spectrum_but">
       <property name="text">
        <string>draw circle spectrum</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="ellipse_time_stat_but">
       <property name="text">
        <string>ellipse time statistics</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QTextEdit" name="textEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="draw_ellipse_spectrum_but">
       <property name="text">
        <string>draw ellipse spectrum</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1278</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
